step-functions/audit_event.asl.json

```diff
@@ -191,12 +191,19 @@
 		},
 		"publish event to TxMa Queue": {
 			"Type": "Task",
-      "InputPath": "$[0]",
-			"Resource": "arn:aws:states:::sqs:sendMessage",
+      "InputPath": "$[0].auditEvent",
+			"Resource": "arn:aws:states:::events:putEvents",
```

Comment:
Can you explain what behaviour change this makes?

Reply:
```						
"Detail.$": "$[0].auditEvent",
"DetailType.$": "$[0].auditEvent.event_name",
```